### PR TITLE
First version of the detailed description of the IDEA drift chamber

### PR DESCRIPTION
**Pull request by BrieucF**
BEGINRELEASENOTES

- Added DriftChamber_o1_v01, a first version of the detailed IDEA drift chamber to enable further technical developments (digitization, tracking, PFlow, ...)

ENDRELEASENOTES

Covered by `test_IDEA_o1_v02`.

Look at `detector/tracker/README.md` to see what should be further improved.

### CODE REVIEW

**Comment by BrieucF** [comment]
The tests are passing now, ready for review

File: detector/tracker/README.md [hunk 1]
```diff
@@ -7,8 +9,24 @@ More details can be found in the talks at the [FCC week 2023](https://indico.cer
...
+
+## DriftChamber
+Detailed (i.e. including wires) description of a unique-volume, high granularity, fully stereo, low mass, cylindrical drift chamber. 
+Originally designed for the IDEA detector, it will be used also for the ALLEGRO concept with different dimensions. 
+A detailed description of the detector it tries to model can be found [here](https://indico.cern.ch/event/932973/contributions/4041314/attachments/2139657/3664808/primavera_FCCworkshop_2020.pdf) or [here](https://indico.cern.ch/event/1283129/contributions/5476695/attachments/2682660/4654170/DeFilippis_DCH_CC.pdf).
 
+### o1_v01
+Very first version of the detailed drift chamber with wires, walls, etc. Taken from an early prototype available in FCCDetectors ([link](https://github.com/HEP-FCC/FCCDetectors/blob/main/Detector/DetFCCeeIDEA/src/DriftChamber.cpp)) and started to debug it, clean it and add the sensitive volume definition.
+It is clearly not finished but made available to enable further technical developments on the digitization, tracking, particle flow, ...
+This version do not rely on any external segmentation, all volumes are defined in the detector builder.
```
Comment (jmcarcell) [inline detector/tracker/README.md:25]:
```suggestion
This version does not rely on any external segmentation, all volumes are defined in the detector builder.
```

File: FCCee/IDEA/compact/IDEA_o1_v02/materials_o1_v01.xml [hunk 2]
```diff
@@ -299,14 +299,6 @@
     <fraction n="0.0925" ref="Ni" />
   </material>
 
-  <!-- FCCeeIDEA: material for the drift chamber -->
-  <material name="GasHe_90Isob_10">
-    <D value="0.0003983999999999999" unit="g/cm3" />
-    <fraction n="0.3826351004462046" ref="He"/>
-    <fraction n="0.011371937371285891"   ref="H" />
-    <fraction n="0.6059929621825095" ref="C" />
-  </material>
```
[the remark this answers is not in the document]
Comment (BrieucF) [reply FCCee/IDEA/compact/IDEA_o1_v02/materials_o1_v01.xml:308]: They are indeed different, the one in this PR comes directly from the original version of the detailed drift chamber implementation in FCCDetector. I have contacted the original author to understand how confident he is in them and will in parallel get in touch with the R&D crew to cross-check. I will update later if needed and added a reminder in the material compact file.

**Comment by BrieucF** [comment]
Thanks Juan!

File: FCCee/IDEA/compact/IDEA_o1_v02/materials_o1_v01.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (BrieucF) [reply FCCee/IDEA/compact/IDEA_o1_v02/materials_o1_v01.xml:308]: I re-computed the material fractions by hand, they are correct in this PR

**Comment by BrieucF** [comment]
Can we merge this?

**Review by andresailer — APPROVED** [review]

**Pull request merged**